embulk-cli/src/main/sh/selfrun.sh

```diff
@@ -44,11 +44,38 @@ while true; do
     esac
 done
 
+embulk_args="$@"
+
+while [ $# -gt 0 ] ; do
+    case "$1" in
+        "-b" | "--bundle")
+            shift
+            export EMBULK_BUNDLE_PATH="$1"
+            shift
+            break
+            ;;
+        *)
+            shift
+            ;;
+    esac
+done
+
+if test -z ${EMBULK_BUNDLE_PATH}; then
+    unset EMBULK_BUNDLE_PATH
+    unset BUNDLE_GEMFILE
+    export GEM_HOME="$(cd && pwd)/.embulk/jruby/$(java -cp $0 org.jruby.Main -e 'print RbConfig::CONFIG["ruby_version"]')"
```

Comment:
@dmikurube FYI
bash on Windows 
* `$HOME` point to `/home/<username>`.
* It's not the same `%HOMEPATH%`
* `%HOMEPATH%` localated at `/mnt/c/Users/<username>`  in bash.

Ref: [Twitter](https://twitter.com/koturn/status/854719055986515968) (Japanese text).